packages/vscode-dependencies-validation/package.json

```diff
@@ -1,49 +1,40 @@
 {
   "name": "vscode-dependencies-validation",
   "displayName": "vscode-dependencies-validation",
+  "version": "1.4.0",
   "private": true,
-  "description": "Displays dependency issues in problems view and marks relevant lines in package.json files",
-  "version": "1.3.2",
-  "license": "Apache-2.0",
-  "publisher": "SAPOSS",
-  "engines": {
-    "vscode": "^1.50.0"
-  },
+  "description": "Diagnostics and quick-fixes for NPM dependencies issues for VSCode",
   "categories": [
     "Other"
   ],
-  "activationEvents": [
-    "*"
-  ],
-  "main": "./dist/extension.js",
+  "engines": {
+    "vscode": "^1.50.0"
+  },
+  "license": "Apache-2.0",
+  "publisher": "SAPOSS",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SAP/app-studio-toolkit",
+    "directory": "packages/npm-dependencies-validation"
+  },
+  "main": "./dist/src/extension",
```

Comment:
main points to compile target rather then bundle target.
this allows more rapid feedback loops while developing.